packages/osd-optimizer/package.json

```diff
@@ -33,7 +33,6 @@
     "pirates": "^4.0.1",
     "postcss": "^8.4.5",
     "rxjs": "^6.5.5",
-    "node-sass": "^8.0.0",
```

Comment:
Why is this moved into `devDependencies`? Was it just originally accidentally included in `dependencies`?

Reply:
Correct